rfcs/text/semantic-release.md

```diff
@@ -0,0 +1,257 @@
+- Start Date: 2019-11-28
+
+# Summary
+
+We, the Carbon team, want to move to a continuous delivery approach. We have identified [`semantic-release`](https://github.com/semantic-release/semantic-release) as a suitable
+choice.
+
+# Basic example
+
+https://semantic-release.gitbook.io/semantic-release/
+
+# Motivation
+
+The number of changes included between `v8` and `v9` were substantial. It has become difficult for users to upgrade due
+to the number of changes. This is due to not releasing fast enough for the throughput of the team.
+
+We want to release faster so users can receive features faster and make small incremental upgrades. This will make
```

Comment:
Maybe this would clarify what we mean?
```suggestion
We want to release more frequently, so users can receive features faster and make small incremental upgrades. This will make
```